src/pages/catalog-page/catalog-options/components/sorting-selects/sorting-selects.tsx

```diff
@@ -0,0 +1,59 @@
+import { messages } from './messages';
+import styles from './sorting-selects.module.css';
+import { catalogStore } from '../../../../../store/catalog-store';
+import { useEffect } from 'react';
+import { observer } from 'mobx-react-lite';
+import { SortField, SortValue, SortOrder } from './enums';
+
+export const SortingSelects = observer(() => {
+  const field = catalogStore.sortField;
+  const order = catalogStore.sortOrder;
+
+  const handleFieldChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const newField = event.target.value;
+    if (newField === SortField.Name_en || newField === SortField.Price) {
+      catalogStore.setSort(newField, SortOrder.Asc);
+    }
+  };
+
+  const handleOrderChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const newOrder = event.target.value;
+    if ((newOrder === SortOrder.Asc || newOrder === SortOrder.Desc) && field) {
```

Comment:
как бы да, но у тебя если !field, то просто селектор скрыт и этот хендлер ты просто вызвать не сможешь. так что можешь просто убрать
```suggestion
    if (newOrder === SortOrder.Asc || newOrder === SortOrder.Desc) {
```

ну или можешь отсавить, если тебе так понятнее. не критично